### ISSUE TITLE
PHPhotoLibrary Support

### ISSUE THREAD

**Issue by mingsai** [hackiftekhar/IQMediaPickerController#15]
The asset could benefit from PHPhotoLibrary Support. I've forked a copy and will see if I can disambiguate the references to the  ALAssetsLibrary. Wish Apple had provided a mediator or bridge object that did this already.

**Comment by hackiftekhar** [comment]
Thank You!

**Comment by hackiftekhar** [comment]
Yes, it's the time to move from ALAssetLibrary to PHPhotoLibrary, but I need to some time to convert it.

**Comment by MahipalV12** [comment]
hii, can you please update project to PHPhotoLibrary.

**Comment by MahipalV12** [comment]
Thanks a lot

i'll update my code
